Demo/UIKit/ComponentsViewController.swift

```diff
@@ -54,39 +54,71 @@ internal final class ComponentsViewController: UIViewController, Presenter {
     // MARK: - DropIn Component
 
     internal func presentDropInComponent() {
-        integrationExample.presentDropInComponent()
+        if componentsView.isUsingSession {
+            integrationExample.presentDropInComponentSession()
+        } else {
+            integrationExample.presentDropInComponent()
+        }
     }
 
     internal func presentCardComponent() {
-        integrationExample.presentCardComponent()
+        if componentsView.isUsingSession {
+            integrationExample.presentCardComponentSession()
+        } else {
+            integrationExample.presentCardComponent()
+        }
     }
 
     internal func presentIdealComponent() {
-        integrationExample.presentIdealComponent()
+        if componentsView.isUsingSession {
+            integrationExample.presentIdealComponentSession()
+        } else {
+            integrationExample.presentIdealComponent()
+        }
     }
 
     internal func presentSEPADirectDebitComponent() {
-        integrationExample.presentSEPADirectDebitComponent()
+        if componentsView.isUsingSession {
+            integrationExample.presentSEPADirectDebitComponentSession()
+        } else {
+            integrationExample.presentSEPADirectDebitComponent()
+        }
     }
 
     internal func presentBACSDirectDebitComponent() {
-        integrationExample.presentBACSDirectDebitComponent()
+        if componentsView.isUsingSession {
+            // TODO: bacs for session
+        } else {
+            integrationExample.presentBACSDirectDebitComponent()
+        }
     }
 
     internal func presentMBWayComponent() {
-        integrationExample.presentMBWayComponent()
+        if componentsView.isUsingSession {
+            integrationExample.presentMBWayComponentSession()
+        } else {
+            integrationExample.presentMBWayComponent()
+        }
     }
 
     internal func presentApplePayComponent() {
-        integrationExample.presentApplePayComponent()
-    }
-
-    internal func requestPaymentMethods() {
-        integrationExample.requestPaymentMethods()
+        if componentsView.isUsingSession {
+            
```

Comment:
Maybe throw alert here?

Reply:
Those are empty to be filled by rest of the team (that one by you :) ), didn't think to do much else

Reply:
Don't feel like merging unfinished code to `develop`, but ok 🤔